flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -66,13 +66,13 @@ public SentinelSequenceNumber toSentinelSequenceNumber() {
 	public static final String STREAM_TIMESTAMP_DATE_FORMAT = "flink.stream.initpos.timestamp.format";
 
 	/** The base backoff time between each describeStream attempt. */
-	public static final String STREAM_DESCRIBE_BACKOFF_BASE = "flink.stream.describe.backoff.base";
```

Comment:
This would be a breaking change. We should leave these properties as is, if they are semantically equivalent.

Reply:
I have retained the properties string values as same but changed the variable names to indicate listShards for readability.